oshdb-api/src/test/java/org/heigit/ohsome/oshdb/api/tests/TestOSHDBFilter.java

```diff
@@ -115,4 +116,20 @@ public void testFilterGroupByEntity() throws Exception {
 
     assertEquals(osmTypeFilterResult, stringFilterResult);
   }
+
+  @Test
+  @SuppressWarnings("ResultOfMethodCallIgnored")
+  public void testFilterNonExistentTag() throws Exception {
+    FilterParser parser = new FilterParser(new TagTranslator(oshdb.getConnection()));
+    try {
+      createMapReducerOSMEntitySnapshot()
+          .filter(parser.parse("type:way and nonexistentkey=*"))
```

Comment:
Is it good to use something which people could actually use, like `nonexistentkey`? Or would something “randomly” generated be better? I assume you checked it with the test database, though. At least `nonexistent` exists in the current OSM dataset ;)

Reply:
> At least nonexistent exists in the current OSM dataset

:rofl: 

Actually, I didn't check it, but made sure that the test fails before applying the fix. I guess it's ok to leave it as is, since we can control the test data used for the unit tests: The keytables of the test data set only contain a handful of tags at the moment.